### PR TITLE
Update version for update 11

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose

$subject

## Checklist
- [ ] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: gradle.properties
Comment (ThisaruGuruge) [inline gradle.properties:1]: Shall we bump to the next minor version?

File: gradle.properties [hunk 1]
```diff
@@ -16,62 +16,62 @@ mySqlDriverVersion=8.0.29
...
+stdlibMssqlDriverVersion=1.7.0-20241126-144300-5155121
+stdlibPostgresqlVersion=1.14.0-20241127-202500-8f2f1cd
+stdlibPostgresqlDriverVersion=1.6.0-20241126-144300-59acbb0
+stdlibDataJsonDataVersion=1.0.0-20241125-114000-0c2f457
 
 # Ballerinax Observe
-observeVersion=1.4.0-20241113-092000-b83ae74
-observeInternalVersion=1.3.1-20241113-101700-265054d
+observeVersion=1.4.0-20241122-184200-c596093
+observeInternalVersion=1.4.0-20241122-185200-9e5b1bb
```
Comment (ThisaruGuruge) [inline gradle.properties:77]:
```suggestion
observeInternalVersion=1.4.0-20241122-185200-9e5b1bb

```

File: gradle.properties
Comment (daneshk) [reply gradle.properties:1]: Fixed in [7d3c670](https://github.com/ballerina-platform/persist-tools/pull/391/commits/7d3c670fb90ea1325e031c3867df733970e8dd17)

File: persist-cli-tests/src/test/resources/test-src/input/tool_test_build_1/Ballerina.toml [hunk 2]
```diff
@@ -32,4 +32,4 @@ filePath = "persist/model.bal"
 [[platform.java21.dependency]]
 groupId = "io.ballerina.stdlib"
 artifactId = "persist.sql-native"
-version = "1.4.0"
+version = "1.4.1"
```
Comment (lnash94) [inline persist-cli-tests/src/test/resources/test-src/input/tool_test_build_1/Ballerina.toml:35]: is this expected to have version `1.4.1` or `1.5.0-SNAPSHOT`?

Comment (daneshk) [reply persist-cli-tests/src/test/resources/test-src/input/tool_test_build_1/Ballerina.toml]: Fixed in [0d19498](https://github.com/ballerina-platform/persist-tools/pull/391/commits/0d1949880afb5ba7cf1949b564f711b368748d87)

**Review by lnash94 — APPROVED** [review]

**Review by ThisaruGuruge — APPROVED** [review]

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**